code/addons/highlight/src/preview.ts

```diff
@@ -8,18 +8,12 @@ const { document } = global;
 
 type OutlineStyle = 'dotted' | 'dashed' | 'solid' | 'double';
 
-export const highlightStyle = (color = '#FF4785', style: OutlineStyle = 'dashed') => `
+const highlightStyle = (color = '#FF4785', style: OutlineStyle = 'dashed') => `
   outline: 2px ${style} ${color};
   outline-offset: 2px;
   box-shadow: 0 0 0 6px rgba(255,255,255,0.6);
 `;
 
-export const highlightObject = (color: string) => ({
```

Comment:
This was a bad port, these styles were not used and highlightStyle shouldn't be exported, we made these changes because they were affecting ESM code generation